bottlecap/src/traces/trace_processor.rs

```diff
@@ -165,7 +168,10 @@ impl TraceProcessor for ServerlessTraceProcessor {
             test_token: None,
         };
 
-        let mut send_data = SendData::new(body_size, payload, header_tags, &endpoint);
+        let send_data_builder = SendDataBuilder::new(body_size, payload, header_tags, &endpoint);
+        let mut send_data = send_data_builder
+            .with_compression(Compression::Zstd(6))
```

Comment:
Can customer set the compression level here for traces?

Reply:
Yeah in a future PR we can add this as a config option